src/instrumentations/utils/PayloadHandler.ts

```diff
@@ -18,13 +17,13 @@
 import { Options } from '../../options';
 import { diag, Span } from '@opentelemetry/api';
 
-export class HttpBodyHandler {
+export class PayloadHandler {
   private maxPayloadSize: number; // The size in bytes of the maximum payload capturing
   private currentBodySize: number; // The size in bytes of the current stream capture size
-  // TODO: maybe add content parsing in the future
+  // TODO: maybe add content encoding parsing in the future
   private totalChunks: any[];
 
-  constructor(options: Options, contentEncoding: string) {
+  constructor(options: Options, contentEncoding?: string) {
     this.maxPayloadSize = options.maxPayloadSize
       ? options.maxPayloadSize
       : 1024;
```

Comment:
I think setting the default value is more suitable within the Options class

Reply:
Yea, im gonna open issue and fix that (there's many code lines like that)